cirq-core/cirq/sim/clifford/stabilizer_state_ch_form.py

```diff
@@ -273,3 +273,32 @@ def project_Z(self, q, z):
             self.omega /= np.sqrt(2)
 
         self.update_sum(t, u, delta=delta)
+
+    def kron(self, other: 'cirq.StabilizerStateChForm') -> 'cirq.StabilizerStateChForm':
+        n = self.n + other.n
+        copy = StabilizerStateChForm(n)
+
+        def diag(a, b):
+            a = np.pad(a, [0, other.n])
+            b = np.pad(b, [self.n, 0])
+            return a + b
```

Comment:
Can do this with a single view-write instead of multi allocate + add + assign.
```python3
copy.G[:self.n, :self.n] = self.G
copy.G[self.n:, self.n:] = other.G
```